tests/integration/conftest.py

```diff
@@ -17,35 +17,43 @@
 #
 
 import os
-
 import pytest
 
+from tests.common import TestUtils
 from tests.integration.cluster_fixture import TrivupFixture
 from tests.integration.cluster_fixture import ByoFixture
 
 work_dir = os.path.dirname(os.path.realpath(__file__))
 
 
+def _broker_conf():
+    broker_conf = ['transaction.state.log.replication.factor=1',
+                   'transaction.state.log.min.isr=1']
+    if TestUtils.use_group_protocol_consumer():
+        broker_conf.append('group.coordinator.rebalance.protocols=classic,consumer')
+    return broker_conf
+
+
 def create_trivup_cluster(conf={}):
     trivup_fixture_conf = {'with_sr': True,
                            'debug': True,
-                           'cp_version': '7.4.0',
-                           'version': '3.4.0',
-                           'broker_conf': ['transaction.state.log.replication.factor=1',
-                                           'transaction.state.log.min.isr=1']}
+                           'cp_version': '7.6.0',
+                           'kraft': TestUtils.use_kraft(),
+                           'version': '3.8.0',
```

Comment:
We should still have a different broker version depending on group protocol, for the consumer group protocol we need a pinned commit on trunk, as some AdminClient changes were merged after 3.8.0.
For the classic one, we can have 3.8.0 to avoid building it.

```suggestion
                           'version': _broker_version(),
```

In librdkafka PR it's using `trunk@3a0efa2845e6a0d237772adfe6364579af50ce18`
```
# This commit corresponds to the DescribeConfigs API change for KIP-848
# KAFKA-14510; Extend DescribeConfigs API to support group configs (#16859)
# https://github.com/apache/kafka/commit/3a0efa2845e6a0d237772adfe6364579af50ce18
# TODO: upgrade to 4.0 when released.
```